addons/sale/views/variant_templates.xml

```diff
@@ -29,24 +29,7 @@
                                     t-att-data-is_single="single"
                                     t-att-data-is_single_and_custom="single_and_custom">
                                     <span t-field="ptav.name"/>
-                                    <span t-if="ptav.price_extra" class="badge badge-pill badge-secondary">
-                                        <!--
-                                            price_extra is displayed as catalog price instead of
-                                            price after pricelist because it is impossible to
-                                            compute. Indeed, the pricelist rule might depend on the
-                                            selected variant, so the price_extra will be different
-                                            depending on the selected combination. The price of an
-                                            attribute is therefore variable and it's not very
-                                            accurate to display it.
-                                            -->
```

Comment:
Well, totally forgot about this comment :D
(Otherwise, I probably wouldn't even have accepted the first PR ^^)